### ISSUE TITLE
About the dataset used in training policy generator

### ISSUE THREAD

**Issue by eruka-w** [accessframework/RAGent#1]
`@click.command()
@click.option('--train_path',
       help='Huggingface dataset name',
       required=True
       )
@click.option('--out_dir', default='../checkpoints/', help='Output directory',show_default=True)
@click.option('--batch_size', default=8, help='Batch size',show_default=True)
@click.option('--lr', default=2e-4, help='Learning rate',show_default=True)
@click.option('--random_seed', default=1, help='Random seed',show_default=True)`
In the training code for the policy generator, it seems that a Hugging Face dataset is required, but the readme doesn't specify which one. Could you please inform me which dataset you are using, or if there's something I'm misunderstanding?

**Comment by accessframework** [comment]
Hi, 

Yes, If you want to train the framework, you have to use the huggingface dataset, which we are planning to release soon!

**Comment by eruka-w** [comment]
May I ask for a few examples of this dataset? Perhaps I can construct it myself and try my own ideas. If possible, thank you very much！！
